src/lib/core/ripple/ripple.ts

```diff
@@ -13,10 +13,14 @@ import {
 import {RippleConfig, RippleRenderer} from './ripple-renderer';
 import {ViewportRuler} from '../overlay/position/viewport-ruler';
 import {RippleRef} from './ripple-ref';
+import {extendObject} from '../util/object-extend';
 
 /** OpaqueToken that can be used to globally disable all ripples. Except programmatic ones. */
 export const MD_DISABLE_RIPPLES = new OpaqueToken('md-disable-ripples');
 
+/** OpaqueToken that can be used to specify a global ripple configuration. */
+export const MD_RIPPLES_CONFIG = new OpaqueToken('md-ripples-config');
```

Comment:
Hm I think that base speed would be a good feature. I will rework the PR. Should we make it a config object then? Or just a `OpaqueToken` for the base-speed (like for disabling ripples)